.travis.yml

```diff
@@ -26,13 +26,15 @@ before_script:
   - |
       if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
       pip install git+https://github.com/SUPERAndroidAnalyzer/travis-cargo.git@fix-coveralls --user &&
-      export PATH=$HOME/.local/bin:$PATH
+      export PATH=$HOME/.local/bin:$PATH &&
+      (travis-cargo install rustfmt || true)
```

Comment:
Please, change this to `travis-cargo install rustfmt` (avoid using `|| true`, since, as I said, it can lead to errors being suppressed.